src/framework/core/js/FluidView.js

```diff
@@ -99,6 +99,7 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
      * @return a single-element jQuery of container
      */
     fluid.container = function (containerSpec, fallible, userJQuery) {
+        var selector = fluid.get(containerSpec, "selector") || containerSpec;
```

Comment:
It's impossible that containerSpec is null - containerSpec.selector || containerSpec is ok here